### ISSUE TITLE
Training resourses and dataset preprocess scripts

### ISSUE THREAD

**Issue by XinyaChen21** [Stability-AI/stable-virtual-camera#36]
Thank you for your outstanding work!

I have a few questions regarding the training setup for your model. Could you share details on the number and type of GPUs required, as well as an estimate of the training duration?

Additionally, do you plan to release the dataset preprocessing scripts? That would be extremely helpful.

Looking forward to your response.

**Comment by jensenz-sai** [comment]
Hi @XinyaChen21,

Thanks for the interest!

- Training resources: As mentioned in this paper, the training takes 100K iters with a batch size of 1472 first with a T=8 model and then further tuned for 600K iters with a batch size of 512. Depending on the hardware and efficiency of the data pipeline, the total GPU hours may vary. Unfortunately, we can't disclose what GPU we used and how long it took us to train.
- Preprocess scripts: We have no current plan to release them. We preprocess all data in a very standard practice. Feel free to drop any questions here, we are here to help.

**Comment by XinyaChen21** [comment]
Thank you for the reply!
I noticed that some datasets have different evaluation splits, e.g. RealEstate10K, CO3D, WildRGB-D, DL3DV-140, Tanks and Temples. Are these splits all subsets of the original test set? Was only the original training set used during training?

**Comment by jensenz-sai** [comment]
Hi @XinyaChen21,

We mostly follow the original split if there is one. If no, we split the train/test splits ourselves and only train on the train split.
